moped-editor/src/views/projects/projectView/ProjectWorkActivity/ProjectWorkActivityTable.js

```diff
@@ -179,14 +205,37 @@ const ProjectWorkActivitiesTable = () => {
     setEditActivity,
   });
 
+  /**
+   * Initialize which columns should be visibile - must be memoized to safely
+   * be used with useHiddenColumnsSettings hook
+   */
+  const defaultHiddenColumnSettings = useMemo(
+    () =>
+      columns.reduce((settings, column) => {
+        settings[column.field] = column.defaultVisible;
+        return settings;
+      }, {}),
+    [columns]
+  );
+
+  const { hiddenColumns, setHiddenColumns } = useHiddenColumnsSettings({
+    defaultHiddenColumnSettings,
+    storageKey: "workActivityTableColumnConfig",
+  });
+
   if (loading || !data) return <CircularProgress />;
 
   return (
     <ApolloErrorHandler errors={error}>
-      <Box sx={{ width: "100%" }}>
+      <Box sx={{ width: "100%", overflow: "auto", minHeight: "700px" }}>
```

Comment:
i set a min height to avoid a really jarring page scroll when the column visibility picker drops down and pushes the bottom of the page with it :/

Reply:
Surprises like this are always fun! 🙃 This seems great and better than a sudden UI shift. 

Also makes me wonder if the Project view should have a minimum height across the all tabs. Not really sure, and maybe the updates to the project list view will help us evaluate blank space elsewhere in the app. 🤔